README.md

```diff
@@ -74,7 +74,7 @@ $ make bootstrap
 ...
 ```
 
-To compile a development version of Vault, run `make` or `make dev`. This will
+To compile a development version of Vault, run `make` or `make dev`. To compile a development version of Vault Enterprise, run `make premdev`. This will
```

Comment:
Should we add a caution that a `premdev` binary will have a perpetual builtin license and thus should be treated with care?

Reply:
Sure, I can do that. Is there anything in particular that we should warn folks about? That is, what are the risks of having a perpetual builtin license?

Reply:
I just mean that the creator handle it with the same care as a license key, making sure not to let it fall into unauthorized hands.